src/tools/nix-dev-shell/shell.nix

```diff
@@ -16,10 +16,15 @@ pkgs.mkShell {
     pkgs.nix
     pkgs.glibc.out
     pkgs.glibc.static
+    pkgs.rust-analyzer
```

Comment:
I feel like this goes towards an aspect of editor configuration that is the users responsibility and not the projects, though I guess having the package around doesn't hurt even when it's unused.